src/Components/NavBar/menuData.ts

```diff
@@ -440,7 +440,7 @@ export const menuData: MenuData = {
       href: "/institutions",
     },
     {
-      text: "Partner with Artsy",
+      text: "Artsy for Galleries",
```

Comment:
@xtina-starr Given we hard-code this in the file above, are we using this data anywhere? Wondering if we should remove it from this file or update the logic above-- what do you think?

Reply:
Yeah I think we can remove it from this file.

Reply:
It looks like there are a few entries that are unused. Would you mind giving it a once-over and removing the ones that aren't necessary as you're working on the desktop-side?

Reply:
Yeah! Definitely.